packages/design/src/colors.css

```diff
@@ -83,20 +83,20 @@
   --color-blue--dark: rgb(1, 25, 35);
 
   --color-taupe: rgb(241, 239, 233);
-  --color-taupe--light: rgb(247, 246, 243);
+  --color-taupe--light: rgb(252, 250, 244);
   --color-taupe--dark: rgb(231, 229, 217);
 
-  --color-green: rgb(50, 130, 28);
-  --color-green--light: rgb(104, 162, 87);
+  --color-green: var(--color-base-green--600);
+  --color-green--light: var(--color-base-green--500);
   --color-green--lighter: rgb(198, 221, 192);
   --color-green--lightest: rgb(241, 247, 238);
   --color-green--dark: rgb(14, 78, 26);
 
-  --color-lime: rgb(153, 194, 71);
-  --color-lime--light: rgb(184, 212, 126);
-  --color-lime--lighter: rgb(214, 230, 181);
-  --color-lime--lightest: rgb(240, 246, 227);
-  --color-lime--dark: rgb(99, 125, 46);
+  --color-lime: var(--color-base-lime--500);
+  --color-lime--light: var(--color-base-lime--400);
+  --color-lime--lighter: var(--color-base-lime--300);
+  --color-lime--lightest: var(--color-base-lime--200);
+  --color-lime--dark: var(--color-base-lime--700);
```

Comment:
@jdeichert this is what I meant by "deprecation layer" / "compatibility mode" - pointing the existing "{color}/dark/light/lighter/lightest" allows us to land this change without breaking any existing usages of the base tokens (including in `@components`) so we can do a migration pass in consuming apps.